streaming/src/main/scala/org/apache/carbondata/streaming/parser/FieldConverter.scala

```diff
@@ -66,30 +65,57 @@ object FieldConverter {
         case bs: Array[Byte] => new String(bs,
           Charset.forName(CarbonCommonConstants.DEFAULT_CHARSET))
         case s: scala.collection.Seq[Any] =>
-          val delimiter = if (level == 1) {
-            delimiterLevel1
-          } else {
-            delimiterLevel2
-          }
+          val delimiter = complexDelimiters.get((level))
```

Comment:
line 68 and line 83 should use uniform style

Reply:
Done